client/utils/base64.c

```diff
@@ -0,0 +1,116 @@
+/*********************************************************************
+* Filename:   base64.c
```

Comment:
Why we need new base64? We already have such functions with address, please find the code, and it's much simpler than this file.

Reply:
Base64 implementation in the address.c class is intended only for conversion from addresses to hashes and vice versa. I can suggest to leave that implementation and reuse it in the address C.
Like 
```
int xdag_address2hash(const char *address, xdag_hash_t hash)
{
	decode_base64(address, hash, 32);

	for (i = 0; i < 8; ++i) {
		*fld++ = 0;
	}

	return 0;
}
```